spec/controllers/searches_controller_spec.rb

```diff
@@ -387,17 +387,19 @@
       end
     end
   end
-
+  
```

Comment:
Remove extra spaces

Reply:
Not sure what space you want me to remove. 
<img width="1038" alt="Screen Shot 2019-07-01 at 5 06 06 PM" src="https://user-images.githubusercontent.com/7727136/60466844-d3ef3080-9c22-11e9-9320-5e81a344f305.png">

Reply:
ahh okay so Rubymine did not highlight it because it was a tab.